sys/kern/vm_map.c

```diff
@@ -425,39 +437,74 @@ void vm_map_dump(vm_map_t *map) {
          (it->prot & VM_PROT_READ) ? 'r' : '-',
          (it->prot & VM_PROT_WRITE) ? 'w' : '-',
          (it->prot & VM_PROT_EXEC) ? 'x' : '-');
-#if 0
-    vm_object_dump(it->object);
-#endif
   }
 }
 
+static vm_map_entry_t *entry_clone_shared(vm_map_t *map, vm_map_entry_t *ent) {
```

Comment:
First argument is unused? Why not full name `vm_map_entry_clone_shared`?

Reply:
done.